src/Contrib/Otlp/_register.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+\OpenTelemetry\SDK\FactoryRegistry::registerSpanExporterFactory('otlp', \OpenTelemetry\Contrib\Otlp\SpanExporterFactory::class);
+\OpenTelemetry\SDK\FactoryRegistry::registerMetricExporterFactory('otlp', \OpenTelemetry\Contrib\Otlp\MetricExporterFactory::class);
+
+\OpenTelemetry\SDK\FactoryRegistry::registerTransportFactory('http/protobuf', \OpenTelemetry\Contrib\Otlp\OtlpHttpTransportFactory::class);
+\OpenTelemetry\SDK\FactoryRegistry::registerTransportFactory('http/json', \OpenTelemetry\Contrib\Otlp\OtlpHttpTransportFactory::class);
+\OpenTelemetry\SDK\FactoryRegistry::registerTransportFactory('http/ndjson', \OpenTelemetry\Contrib\Otlp\OtlpHttpTransportFactory::class);
```

Comment:
Transport should be just `http`.

Reply:
done, I've updated `TransportFactory` to deal with this, rather than burden the various factories with stripping the content type off the protocol...